.github/workflows/rust.yml

```diff
@@ -26,13 +26,13 @@ jobs:
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2020-06-09
+          toolchain: nightly
           override: true
-      - name: Install wasm-pack
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: wasm-pack --version ${{ env.wasmpackVersion }}
+#      - name: Install wasm-pack
```

Comment:
Why is it commented out?

Reply:
My mistake. Debugging. Will comment back in.